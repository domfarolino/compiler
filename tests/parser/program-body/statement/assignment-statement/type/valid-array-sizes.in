program dominic is

  string target[0: 12];
  string expression[24: 36];

begin

  target := expression;

end program.
