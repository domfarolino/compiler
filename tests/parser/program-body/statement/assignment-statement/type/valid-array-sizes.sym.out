+--------Symbol table--------+
   string      expression[24 : 36] isGlobal:     -- paramType:  None
   string          target [0 : 12] isGlobal:     -- paramType:  None
+--------End symbol table----+

Program compiled successfully
[1;31m[0m